chapters/synchronous.tex

```diff
@@ -1096,43 +1096,46 @@ \section{Clocked Discretized Partition}\label{continuous-time-equations-in-clock
 have only a value at a clock tick. Therefore, such a partition is
 handled in the same way as any other clocked partition. Especially,
 operators such as \lstinline!sample!, \lstinline!hold!, \lstinline!subSample! must be used to communicate
-signals of the discretized continuous-time partition with other
-partitions. Hereby, a discretized continuous-time partition is seen as a
+signals of the discretized partition with other
+partitions. Hereby, a discretized partition is treated similarly as a
```

Comment:
In the previous sentence, it sounds as if a discretized partition is a kind of clocked partition.  Here it sounds as if they are not.

Reply:
There are a couple of more places as well in earlier paragraphs of this section that also need to be changed in order to avoid calling a discretized partition a clocked partition.

Reply:
That has now been resolved.

Reply:
OK.  Let's start over from the new text.  Resolving.